session_0/fizz_buzz.rb

```diff
@@ -36,6 +38,26 @@
 #
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
+
 def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
+  if x<1 or y<1 or n<0
```

Comment:
What if x is divisible? Is `%` operator defined for zero?

Reply:
Didn't got the question properly like I have defined the case of x = 0 so if x=0 occurs it will raise an Argument Error. Also, for X%Y, % operator is not defined when Y == 0:

Reply:
@abhishekkumar2718 can u review my answers?

Reply:
Right, your answer is correct. I missed the initial code

Reply:
ohk..cool